templates/dag-deploy/dag-deploy-rolebinding.yaml

```diff
@@ -34,6 +34,11 @@ subjects:
 - kind: ServiceAccount
   name: {{ include "astro.scheduler.serviceAccountName" . }}
   namespace: "{{ .Release.Namespace }}"
+{{- if .Values.airflow.dagProcessor.enabled }}
+- kind: ServiceAccount
+  name: {{ include "astro.dagProcessor.serviceAccountName" . }}
+  namespace: "{{ .Release.Namespace }}"
+{{- end }}
```

Comment:
should we have this dagProcessor feature flagged or lets enable this sa whether or not feature enabled